DiscordBot/Settings.job

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
What's this used for? 🙂

Reply:
It's a redundant config file. Removed c98cdf6 🙂